README.md

```diff
@@ -147,95 +152,137 @@ MODEL_NAME = "user123 customer churn model"
 
 Continue to run the remaining cells in the section to save the model to Cloud Pak for Data. We'll be able to test it out with the Cloud Pak for Data tools in just a few minutes!
 
-## 6. Create a project release
-
-Next, we'll create a project release and tag the model under version control. We'll use model management and deployment to make the released model available as a web service (REST API).
-
-### Commit the project changes
 
-On the project home click on the *Git* button on the top row and choose *Commit*
+## 6. Create a Space for Machine Learning Deployments
 
-![Commit the project changes](doc/source/images/project-1-git-commit.png)
+Go the (☰) menu and click `Analyze` -> `Analytics deployments`:
 
-A list of the assets will appear that were created in this project. Provide a commit message to identify the changes being pushed.
+![(☰) Menu -> Analytics deployments](doc/source/images/ChooseAnalyticsDeployments.png)
 
-![Provide a commit message](doc/source/images/project-2-git-commit-message.png)
+Click on `+ New deployment space):
```

Comment:
Looks like an open back-tick on this line

Reply:
done